apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/admin/api/client/BftScanConnection.scala

```diff
@@ -870,6 +870,7 @@ object BftScanConnection {
     protected val initialScanConnections: Seq[SingleScanConnection]
     protected val initialFailedConnections: Map[Uri, Throwable]
     protected val connectionBuilder: Uri => Future[SingleScanConnection]
+    protected val persistScanUrlCallback: Option[Seq[DsoScan] => Future[Unit]]
```

Comment:
```suggestion
    protected val refreshScanUrlsCallback: Option[Seq[DsoScan] => Future[Unit]]
```

the fact that we persist them as part of that callback doesn't matter here

Reply:
Resolved: Used refreshScanUrlsCallback